src/test/java/edu/oakland/test/display01/SatelliteSignalCheckRequestUnitTests.java

```diff
@@ -9,7 +9,7 @@
 import org.junit.jupiter.api.Test;
 
 @DisplayName("Sattelite Signal Check Request Tests")
-public class SatelliteSignalCheckRequestTests {
+public class SatelliteSignalCheckRequestUnitTests {
```

Comment:
The class name was fine. That should be changed back.
```suggestion
public class SatelliteSignalCheckRequestTests {
```